ui/app/pages/swaps/fee-card/index.scss

```diff
@@ -56,6 +56,29 @@
     }
   }
 
+  &__info-tooltip-container {
+    height: 10px;
+    width: 10px;
+    justify-content: center;
+
+    > div {
```

Comment:
rather than relying on tag names I think I would prefer a classname be added to those `p` tags (if suggestion is taken, `div` if not)